src/main/java/com/coveo/pushapiclient/CatalogSource.java

```diff
@@ -0,0 +1,17 @@
+package com.coveo.pushapiclient;
```

Comment:
can't we put the new classes in a new package ?

Reply:
If we do that, where should we expose classes shared by both packages? E.g.`DocumentBuilder`, `PlatformUrl`, `UserSecurityIdentityBuilder`...